STAR-Delegation/Makefile

```diff
@@ -14,11 +14,13 @@ ifeq ($(strip $(svgcheck)),)
 $(error svgcheck not found. To install svgcheck: 'pip install svgcheck')
 endif
 
-kdrfc_check := $(shell gem list --installed kramdown-rfc2629 -v '>= 1.3.3')
-ifeq ($(kdrfc_check), true)
-kdrfc := kdrfc -3hp
-else
-$(error kdrfc not at the right version. To install it: 'gem install kramdown-rfc2629 -v '>= 1.3.3'')
+ifeq ($(strip $(kdrfc)),)
```

Comment:
This is a small refactoring to allow callers to supply their own `kdrfc` -- consistent with the `?=` philosophy.